### ISSUE TITLE
Accessibility feedback for Usability Testing Prep - eBenefits [CH36]

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#14264]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Point of Contact

**VFS Point of Contact:** Jennifer

## Details

This ticket will container contains Accessibility feedback for Usability Testing Prep - eBenefits [CH36].

> Note: As this usability testing is complete, I will provide guidance as comments on this issue.

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

**Comment by jenstrickland** [comment]
For recruiting, since the audience specifically has **service-connected disabilities** it would be appropriate to ensure that the participants all have service-connected disabilities.

> The Chapter 31 program (CH31), Veteran Readiness and Employment (VR&E), provides counseling and planning to eligible Veterans and service members **with service-connected disabilities**…

**Comment by jenstrickland** [comment]
Before this work moves into development, please schedule a [VSA Dev Intent Review](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/accessibility/vsa-accessibility-review-process.md#development-intent-review) to ensure semantic markup and any design adjustments are included — or documented for the backlog.

**Comment by jenstrickland** [comment]
The confirmation screen **should** match other confirmation screens.

![Screen Shot 2020-10-01 at 2 08 14 PM](https://user-images.githubusercontent.com/57469/94846700-9104a580-03ef-11eb-8edc-64f60fcdc3b7.png)


![image (10)](https://user-images.githubusercontent.com/57469/94846799-b98c9f80-03ef-11eb-97d9-c96ab42e278f.png)

**Comment by jenstrickland** [comment]
@1Copenut — ready for VSP review

**Comment by 1Copenut** [comment]
I have just two additional **consider** items:

1. The Print this screen button is clickable in the UXPins, but doesn't link to a new screen. It might be worth adding that view if there's time, or removing the clickable hot spot.
2. Consider matching up the numbered steps with the progress bar "counter". Screenshots below.

No further feedback.

---
![Screen Shot 2020-10-01 at 12 52 28 PM](https://user-images.githubusercontent.com/934879/94861443-28bcc080-03fd-11eb-9414-2b2ea68fd1b3.png)

---
![Screen Shot 2020-10-01 at 12 52 36 PM](https://user-images.githubusercontent.com/934879/94861457-2ce8de00-03fd-11eb-8f96-70a541e1a65f.png)

**Comment by jason-gcio** [comment]
This should be addressed and/or completed
